packages/lib-subject-viewers/test/setup.js

```diff
@@ -0,0 +1,43 @@
+import chai from 'chai'
+import dirtyChai from 'dirty-chai'
+import sinonChai from 'sinon-chai'
+import { JSDOM } from 'jsdom'
+
+chai.use(dirtyChai)
+chai.use(sinonChai)
+global.expect = chai.expect
+
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>', { url: 'https://localhost' })
+const { window } = jsdom
+
+function copyProps (src, target) {
+  const props = Object.getOwnPropertyNames(src)
+    .filter(prop => typeof target[prop] === 'undefined')
+    .map(prop => Object.getOwnPropertyDescriptor(src, prop))
+  Object.defineProperties(target, props)
+}
+
+class ResizeObserver {
+  disconnect () {}
+  observe () {}
+  unobserve () {}
+}
+
+class IntersectionObserver {
+  disconnect () {}
+  observe () {}
+  unobserve () {}
+}
```

Comment:
These do not need to be included until you are using these APIs in the viewer.

Reply:
Removed